package-lock.json

```diff
@@ -2342,15 +2317,15 @@
       "dependencies": {
         "@oclif/config": {
           "version": "1.12.10",
-          "bundled": true,
```

Comment:
@abernix what's the verdict on this `bundled/resolved` flapping? I remember you mentioning it, but I don't know if there was resolution. Something with a new version of node maybe?

Reply:
Switching from `"bundled": true` to `resolved` is great.  The key is make sure you're not committing anything that adds a new `"bundled": true`.  This particular one was _added_ by Renovate, but due to a bug in `npm`.  Renovate is trying to get it ironed out while the npm bug still exists.  See this comment: https://github.com/renovatebot/renovate/issues/3436#issuecomment-475930530

Anyhow, this LGTM.

Reply:
Glad to hear it, and thanks for the info @abernix!